src/main/java/Ui.java

```diff
@@ -0,0 +1,46 @@
+public class Ui {
+    private TaskList taskList;
+
+    public Ui (TaskList taskList) {
```

Comment:
The UI class shouldn't have to be instantiated with a tasklist given that operations will occur on the tasklist and the tasklist being used in the UI class won't contain the latest information